packages/design-system-dashboard-cli/test/search-files.spec.js

```diff
@@ -0,0 +1,76 @@
+const memfs = require('memfs');
+const {
+  readAllModules,
+  readFile,
+  readAllTemplates,
+  search,
+} = require('../src/search-files');
+
+const env = {
+  "repos": {
+    "vets-website": "mock-vw/",
+    "content-build": "mock-cb/"
+  }
+}
+const jsxFile = 'const phone = () => (<va-telephone contact={phoneContact} />)';
+
+const json = {
+  '../.env.json': JSON.stringify(env),
+    'mock-vw/jsx-file.jsx': jsxFile,
+    'mock-vw/js-file.js': 'some JS',
+    'mock-vw/txt-file.txt': 'Some text', 
+    'mock-cb/liquid-template.liquid': 'A template',
+    'mock-cb/html-layout.html': 'HTML page',
+};
+
+jest.mock('fs');
+memfs.vol.fromJSON(json, '.');
+
+describe('readAllModules', () => {
+  it('reads only js and jsx files', () => {
```

Comment:
I wanted to add a test showing that it excluded `*.unit.jsx` files since that's what the pattern _looks_ like it's doing:

https://github.com/department-of-veterans-affairs/component-library/blob/ffe9df32efc94776cb7141120afa5725c14a61fb/packages/design-system-dashboard-cli/src/search-files.js#L21-L22

But if I add a `'mock-vw/js-file.unit.jsx'` key to the JSON up above, that file gets included in this test. Once we have good coverage to prevent regressions we may want to modify code so that it actually does what we think it does.